src/models/Proxy.js

```diff
@@ -0,0 +1,40 @@
+const Joi = require('joi');
+const BaseModel = require('./BaseModel');
+
+const schema = {
+	id: Joi.number().integer().min(1).required().disallow(null),
```

Comment:
I think `.disallow(null)` is not needed anywhere, it shouldn't accept null by default.